sdk/evaluation/azure-ai-evaluation/setup.py

```diff
@@ -0,0 +1,88 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import os
+import re
```

Comment:
Need to add the following alongside the `setup.py`:

- A proper [`MANIFEST.in`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/MANIFEST.in)
- Add a `py.typed` [configuration](https://github.com/Azure/azure-sdk-for-python/blob/c153e1ccbd15dbf717027eac3b6537123462c163/sdk/core/azure-core/setup.py#L67)
- An `sdk_packaging.toml` file with content:
  ```
  [packaging]
  auto_update = false
  ```
  So our auto formatting doesn't mess with your package.

If you don't want to enable all the checks you're going to need to create a `pyproject.toml` disabling the individual checks that you don't want to deal with right at the moment.

Reply:
I've added those in. Can you take a look again @scbedd?